src/UndockedRegFreeWinRT/Nuget/Microsoft.Windows.UndockedRegFreeWinrt.targets

```diff
@@ -1,39 +1,43 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup Condition="'$(URFW_Configuration)' == ''">
+    <URFW_Configuration>$(Configuration)</URFW_Configuration>
```

Comment:
could provide failsafe defaults here:
```
<URFW_Configuration Condition="'$(Configuration)' == 'Debug'">Debug</URFW_Configuration>
<URFW_Configuration Condition="'$(Configuration)' != 'Debug'">Release</URFW_Configuration>
```

Reply:
the problem is I think this was causing problems with some projects. I had something before that didn't require an opt-in but it was failing tests. I'll explore just a bit more to see if I can figure out what was going wrong in that case.